contrib/python-bindings/source/triangulation_wrapper.cc

```diff
@@ -808,11 +810,14 @@ namespace python
       else if (format.compare("vtk") == 0)
         output_format = GridOut::OutputFormat::vtk;
       else if (format.compare("vtu") == 0)
-        output_format = GridOut::OutputFormat::vtu;
+        {
+          output_format = GridOut::OutputFormat::vtu;
+          GridOutFlags::Vtu flags(true);
```

Comment:
Would you be willing to write this as
```suggestion
          GridOutFlags::Vtu flags(/* serialize_triangulation= */true);
```
to make the intent clear?